plugins/javadoc/src/test/kotlin/org/jetbrains/dokka/javadoc/JavadocIndexTest.kt

```diff
@@ -69,14 +70,30 @@ internal class JavadocIndexTest : AbstractJavadocTemplateMapTest() {
         }
     }
 
+    @Tag("onlyDescriptors") // K2: does not have getters: getName, getOrdinal
```

Comment:
The comment explains the code (which is the more obvious part), but it doesn't explain the intention and reasoning behind it. Why doesn't K2 have getters for name and ordinal? Is this expected? Will it be fixed in the future? If this is a bug of K1, maybe the comment should be about that instead will be less confusing: "K1 mistakenly generates getName and getOrdinal functions"

It looks like this test can also benefit from what I proposed above about descriptor-only or symbol-only tests with different asserts, as right now this test is simply muted for K2 and it's unclear if it will ever be unmuted